topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,383 @@
+>      - *"Van der Waals scale"*: `0.25` Sets the minimum spacing between generated muons and existing atoms
+>      - *"Output folder name"*: `muon-airss-out` Name to call the output folder, only relevant when using the zipped intermediary outputs of the tools
+>      - *"Geometry optimization tolerance"*: `0.02` Tolerance on geometry optimisation in eV/AA
+>    - *"Calculator parameters"*:
+>      - *"Optimization calculator"*: `UEP` Method to use for optimisation
+>      - *"Gaussian width factor"*: `4.0` Adjusts how atom potentials are modelled
+>    - *"Clustering Parameters"*:
+>      - *"Clustering method"*: `hierarchical` Perform hierarchical clustering (other option is k-means)
+>      - *"t parameter for hierarchical clustering"*: `0.2` Threshold value for clusters
+>      - *"Supercell"*: `1` Supercell size and shape to use
+>      - *"K-points grid"*: `[1,1,1]` List of three integer k-points
+>      - *"Max SCC steps"*: `200` If applicable, max number of SCC steps to perform
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `Configuration for si` (Output dataset)
+>
+>    > <comment-title>Optimization calculator</comment-title>
+>    >
+>    > Here we chose {UEP} as our method for the optimisation (step 3), as
+>    > described earlier. The other alternatives are CASTEP and DFTB+ Each
+>    > calculator will have different settings associated with it. DFTB+ has a 
+>    > corresponding tool, {% tool [PyMuonSuite AIRSS DFTB+ Optimise](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/pm_dftb_opt/pm_dftb_opt/0.1.1) %}
+>    > that could be used for stage 3, however CASTEP does not. Instead,
+>    > if we had used CASTEP we would have to manually run CASTEP on these
+>    > structures outside of Galaxy, then zip and upload the results to use as
+>    > the input to step 4.
+>    {: .comment}
+>
+>    > <comment-title>Clustering method</comment-title>
+>    >
+>    > The choice here is between `hierarchical` and `k-means`. `hierarchical`
+>    > clustering builds a tree branching progressively from a single "trunk"
+>    > containing all the stopping sites to multiple "leaves" of each individual
+>    > site. The number of clusters is determined by the t parameter: `0`
+>    > corresponds to individual sites, with the number of clusters decreasing
+>    > as t increases towards `1`. `k-means` attempts to create clusters which
+>    > have equal variance within them. In this case the number of clusters to
+>    > (attempt to) form is the available parameter.
+>    {: .comment}
+>
+{: .hands_on}
+
+The output of this tool is a simple text file containing all the parameters we
+set, formatted for PyMuonSuite. As an alternative to running this step, an
+existing configuration file (either from an earlier use of the tool or uploaded
+directly) can be used as the input for the other PyMuonSuite tools.
+
+## AIRSS Stage 2: Generate muonated structures
```

Comment:
This title is the same as the one above, Eli's suggestion was
https://github.com/muon-spectroscopy-computational-project/training-material/pull/4#discussion_r1064570890

Reply:
Although just seen the one below is labelled as Optimisation instead?

Reply:
Introducing the configuration file (should have) bumped all the stages up by one. This is correct but stage 1 should be renamed to something about the configuration file. Thanks for noticing this.